templates/branded-content-example.html

```diff
@@ -0,0 +1,136 @@
+		<p>After these tests, Jonna compared the fingerprints for the ancient amber with those obtained in previous
+			studies for modern-day resins. Five of her six samples matched a known type of amber. It is what scientists call “Group A.” Those bits of
+			amber likely came from conifers, or cone-bearing trees, that belong to a group called Aracariauaceae (AIR-oh-kair-ee-ACE-ee-eye).
+			Found almost worldwide during the dinosaur era, these thick-trunked trees now grow mainly in the Southern
+			Hemisphere.</p>
+		<blockquote>Were some earlier magnetar flares misdiagnosed?</blockquote>
+		<p>Trees have been around for some 370 million years. These long-lived woody plants can be found on every
+			continent except Antarctica. Scientists have identified more than <strong>60,000 tree species</strong>. And
+			they’ve estimated that there are <strong>3.04 trillion</strong> individual trees growing across the globe.</p>
+		<p>Trees provide homes for animals and shade for people on hot days. One of their big benefits is that they soak
+			up and store the greenhouse gas carbon dioxide. Some people have suggested that planting trees could help to
+			negate the impact of global warming. Researchers last year, though, warned that the easy and simple idea
+			wasn’t so easy and simple after all. It would be difficult to plant enough trees. However, planting trees could help —
+			if done thoughtfully, and in the right places.</p>
+		<img class="fc-img" src="https://source.unsplash.com/phIFdC6lA4E/435x230">
+		<p>Second, planets have to be big enough to have enough gravity to force them into a round shape.
+			Finally, a planet also needs to have enough gravity to keep other objects — such as asteroids — out of its path around its star.
+			<strong>In other words, planets are selfish</strong> — they don’t share their orbit with any other objects.</p>
+		<p>The word “planet” comes from the Greek word “planetes.” It means “wanderer.” That’s because when seen from Earth,
+			planets move around a lot more in the sky than stars do. Even with the current definition of a planet, they aren’t rare.
+			In fact, there are more planets in our galaxy than there are stars.</p>
+		<p>Astronomers had seen flaring magnetars in the Milky Way. But they were so bright that it was impossible to get
+			a good enough look at them and measure their brightness. <strong>glimpses of flaring magnetars</strong> in other galaxies
+			may have been spotted before, too. But “the others were all a little circumstantial,” says Victoria Kaspi. They were
+			“not as rock solid” as the newfound one, she says. Kaspi is astrophysicist at the McGill Space Institute in Montreal, Canada.
+			She was not involved in the new discovery. “Here you have something that is so incontrovertible,” she says. “It’s like, okay, this is it. There’s no question anymore.”</p>
+	</article>
+
+	<div id="flowcards-slot" style="height: 30px; background-color: yellow;"></div>
```

Comment:
This was part of a test, this element should never be visible 😅
```suggestion
	<div id="flowcards-slot"></div>
```

Reply:
yeah I imagined, but didn't really change much here since the html is going to be changed anyway

Reply:
But this file is not going to change, no? I mean, we're going to have a new HTML for the flowcard content but there is no need to create a new template for the demo.

Btw, I forgot to mention, I'd create a new folder for this experience at `experiences/branded-content` and move the template there. WDYT?

Reply:
ah okey, I though that there would be new templates for both 👌 

Yes I'll put them in a folder

Reply:
Actually, regarding the template, wouldn't you keep it here and add the other template (the flowcard content) in the folder `experiences/branded-content` ?

Reply:
Since this template will be used for the branded content experience, I'd move it to the experience folder too.